src/osg/TextureCubeMap.cpp

```diff
@@ -293,12 +293,8 @@ void TextureCubeMap::apply(State& state) const
             _textureWidth = _textureHeight = minimum( _textureWidth , _textureHeight );
         }
 
-        GLenum texStorageSizedInternalFormat = extensions->isTextureStorageEnabled && (_borderWidth==0) ? selectSizedInternalFormat(_images[0].get()) : 0;
```

Comment:
think this whole file can go back to the 3.6 version now that selectSizedInternalFormat is going back into Texture.cpp